### PR TITLE
no issue - artwork process additions

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: _pages/images_guide.md [hunk 1]
```diff
@@ -23,11 +23,24 @@ Add content (size, filetype, naming convention)
...
+
+### Requesting a diagram
+
+1. Submit a request for a diagram at the following link: https://docs.google.com/forms/d/e/1FAIpQLSdF8V0n3E-aAfRen8vZchXCxwA0iDTd0QYyASnutznfGLATvA/viewform.
+
+2. Generally, allow about 4 days for a single diagram. If you submit a few, it will likely take more time.
+
+3. Review the artwork and provide comments. They can also work with an SME, when needed. 
+
+4. When they complete your artwork, the design team will send you a final copy of the diagram in PNG format for you to add to our *images* folder.
```
Comment (dockerymick) [inline _pages/images_guide.md:36]: When the design team completes your artwork, they send a final copy of the diagram in PNG format...

File: _pages/images_guide.md [hunk 2]
```diff
@@ -23,11 +23,24 @@ Add content (size, filetype, naming convention)
...
 ### When to use diagrams
 
-Add content
+Diagrams include flow charts, and artwork that use shapes and objects to show relationships and transfer of information.
+
+### Requesting a diagram
+
+1. Submit a request for a diagram at the following link: https://docs.google.com/forms/d/e/1FAIpQLSdF8V0n3E-aAfRen8vZchXCxwA0iDTd0QYyASnutznfGLATvA/viewform.
+
+2. Generally, allow about 4 days for a single diagram. If you submit a few, it will likely take more time.
```
Comment (dockerymick) [inline _pages/images_guide.md:32]: it is likely to take more time.

**Review by dockerymick — APPROVED** [review]
few comments, thanks for handling this Chris!

**Review by swopebe — APPROVED** [review]

**Comment by swopebe** [comment]
Thanks for adding this--so important to have process down and documented. This helps a lot!

File: _pages/images_guide.md (hunk 1 above)
Comment (swopebe) [reply _pages/images_guide.md:36]: I think it's important to talk about how to save the file, where to save it, how to test it in preview, etc....

**Pull request merged**